dangerzone/logic.py

```diff
@@ -18,9 +18,9 @@
 log = logging.getLogger(__name__)
 
 
-class GlobalCommon(object):
+class DangerzoneCore(object):
     """
-    The GlobalCommon class is a singleton of shared functionality throughout the app
+    Singleton of shared state / functionality throughout the app
```

Comment:
`Dangerzone{Core,Gui}` is basically a collection of common functions and state for:

* Graphics (terminal colors, fonts, window icon)
* Settings (app path, OCR languages)
* Qt app
* Docker status

The fact that it's used for so many different stuff makes it candidate for becoming bloated, and not easily shareable between contexts. I understand that this class (and the common ones before it) act as an easy way to share the above logic between different contexts, but I believe we would greatly benefit if we moved the above logic into their own modules. We would be able to unit test them, and use them from any context, without having to pass around the Dangerzone class.

Note that we definitely need a "context-like" class, which is used to store the state for the conversion operation as a whole, but that's a separate thing. Also, in this PR I don't see this class holding any of that state, which is why I'm still not sure why it's needed. I believe it's used in subsequent PRs, so in this case, it would be more clear if we explain why we want it in the commit message.

Reply:
Understandable. I'm bringing it up now so that we don't overload this class in subsequent PRs. But we can see that later on.